test/runSaga.js

```diff
@@ -67,5 +63,13 @@ test('runSaga', assert => {
   )
 
   task.done.catch(err => assert.fail(err))
-
 })
+
+test('put causing sync dispatch response in store-like subscriber', assert => {
+  const reducer = (state, action) => action.type
+  const store = storeLike(reducer, {})
+
+  const { rootSaga, runTest } = syncDispatchTest(assert, store)
```

Comment:
i didnt do that because of the API differences, I've changed it to to accept 3rd parameter which is a callback receiving a saga from the `syncDispatchTest ` and need to run a saga with appropriate API